requirements-freqai-rl.txt

```diff
@@ -0,0 +1,8 @@
+# Include all requirements to run the bot.
+-r requirements-freqai.txt
+
```

Comment:
will we need a dedicated image? (like `:develop_freqai_rl`) - or shall we add these requirements to the regular freqAI image? (which will probably inflate the size of that image by whatever space the below requirements take...).